### PR TITLE
fix: bugs from #1066

### PR DESCRIPTION
**Pull request by Rudomitori**
Fix the first and third bugs described in #1077

### CODE REVIEW

**Review by shocklateboy92 — DISMISSED** [review]
LGTM.

Do you want to merge this as is, and fix the second bug mentioned in #1077 in a separate PR?

Or should I wait for you to push the fix for the second one also into this branch?

File: Src/CSharpier/SyntaxPrinter/ArgumentListLikeSyntax.cs [hunk 1]
```diff
@@ -10,14 +10,15 @@ FormattingContext context
     )
     {
         var docs = new List<Doc> { Token.Print(openParenToken, context) };
+        var lambdaId = Guid.NewGuid();
 
         switch (arguments)
```
Comment (shocklateboy92) [inline Src/CSharpier/SyntaxPrinter/ArgumentListLikeSyntax.cs:15]: I forgot to mention this in your original PR, but this all branches of this switch statement just add something to the list.
Wouldn't it look better if you used the switch expression instead?

Comment (Rudomitori) [reply Src/CSharpier/SyntaxPrinter/ArgumentListLikeSyntax.cs]: The expression looks almost the same for me

**Comment by Rudomitori** [comment]
I will fix the second bug in other PR

**Comment by Rudomitori** [comment]
Well, I've fixed the last bug before PR is merged, so I've decided to add the changes in this PR

**Review by shocklateboy92 — APPROVED** [review]
Cheers!

**Pull request merged**